src/content/docs/security/security-privacy/compliance/hitrust.mdx

```diff
@@ -132,11 +146,11 @@ HITRUST doesn't provide certification for the following services:
       </td>
 
       <td>
-        AWS
+        N/A
```

Comment:
@jxainguyen similar to my question in the other doc, does it make sense to replace the N/A in the second column with this: AWS, Azure, and GCP?